src/cli.js

```diff
@@ -1,8 +1,146 @@
 #!/usr/bin/env node
 
-require('yargs')
-    .commandDir('commands')
-    .demandCommand()
-    .strict()
-    .help()
-    .argv;
+const decamelize = require("decamelize");
+const yargs = require("yargs");
+
+const github = require("./index");
+
+/**
+ * We probably don't want to require this dynamically as it creates a dependency
+ * on the internal implementation of `@octokit/rest`. I can't see a way to access
+ * these route definitions through any of Octokit's public methods, so we should
+ * possibly just extract and commit the `routes.json` file to this package whenever
+ * we update the `@octokit/rest` dependency.
+ */
```

Comment:
On the flipside, would it be better for the build to break if `octokit` changes their `routes.json`? And maybe a test somewhere to check that `@octokit/rest/plugins/rest-api-endpoints/routes.json`

"extract and commit the `routes.json` file to this package whenever we update the `@octokit/rest` dependency" might be easy to miss when upgrading octokit.

Renovate would also make it more explicit, what version of octokit is used.

Could work either way though.

Reply:
Yeah, this definitely needs more thinking through. I might even raise an issue on the Octokit repository asking if they'd be willing to consider exposing the routes through a public `octokit.whatever()` method.